cmd/alertmanager-bot/main.go

```diff
@@ -117,6 +119,11 @@ func main() {
 		Default("/templates/default.tmpl").
 		ExistingFilesVar(&config.templatesPaths)
 
+	a.Flag("parse.mode", "The template formatting mode HTML/Markdown").
+		Envar("TEMPLATE_FORMAT").
+		Default(string(telebot.ModeHTML)).
```

Comment:
I used `telebot.ParseMode` because `telebot.SendMessage()` depends on it, we can do what you said above, but still, I need to cast the param in `telebot.SendMessage()` to `telebot.ParseMode` to be able to compile cf:
```
pkg/telegram/bot.go:320:85: cannot use b.parseMode (type ParseMode) as type telebot.ParseMode in field value
make: *** [build] Error 2
```
What do you think? casting casting still holds the dependancy anyway.